docs/getting-started/advanced-installation.md

```diff
@@ -0,0 +1,83 @@
+---
+id: advanced-installation
+title: Advanced Installation
+---
+
+This is only for users that have basic understanding on cloud infrastructure and Kubernetes. 
+If you are not experienced with cloud configuration, please check our [installation guide](installation.md) instead. 
+
+## Prerequisites
+
+Please follow our [installation guide](installation.md) to install [Argo Workflow](installation.md#install-argo-workflow) and [Cert Manager](installation.md##install-cert-manager) instead.
+If you are not planning to support SSL certificates, please omit the installation for `Cert Manager`.
+
+
+## Install KintoHub
+
+Run the following steps depending on the setup you want.  
+Each one of the following steps are "incremental", you must run all the steps prior to the setup your chose.
+
+### Minimum Configuration
+
+This will install KintoHub with minimum settings, which is on a local cluster with no inbound internet connection.
+
+```sh
+export KINTO_ARGS="--set minio.resources.requests.memory=null \
+--set minio.makeBucketJob.resources.requests.memory=null \
+--set builder.env.IMAGE_REGISTRY_HOST=kintohub \
+--set builder.workflow.docker.registry=https://index.docker.io/v1/ \
+--set builder.workflow.docker.email=devaccounts@kintohub.com \
+--set builder.workflow.docker.username=docker-username \
+--set builder.workflow.docker.password=docker-password"
+```
+
+### Enable Public Access to Deployed Services 
+
+Adding the following arguments will expose the web services deployed publicly.
+
+```sh
+export KINTO_ARGS="${KINTO_ARGS} \
+--set nginx-ingress-controller.service.type=LoadBalancer \
+--set common.domainName=oss.kintohub.net"
```

Comment:
I thought u wanted to do some `{variable}` to make sure they change the variables